src/test/java/com/keyboardbuilder/enterprise/dto/KeyboardCaseTest.java

```diff
@@ -0,0 +1,48 @@
+package com.keyboardbuilder.enterprise.dto;
```

Comment:
3. JUnit test for Keyboard case. Follows the same ideas as the previous test. Uses Given-When-Then format for Junit tests.